bot/exts/evergreen/battleship.py

```diff
@@ -142,7 +142,7 @@ def get_square(grid: Grid, square: str) -> Square:
         """Grabs a square from a grid with an inputted key."""
         index = ord(square[0]) - ord("A")
         number = int(square[1:])
-
+        index.upper()
```

Comment:
Index here is a number, you need to uppercase the letter before `ord()` gets called on it in line 143.